compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -427,8 +427,8 @@ def submit(self, fn, *args, **kwargs):
             a ``.result()`` when the Globus Compute web services receive and stream it.
         """
         if self._stopped:
-            err_fmt = "%s is shutdown; no new functions may be executed"
-            raise RuntimeError(err_fmt % repr(self))
+            err = f"{self!r} is shutdown; no new functions may be executed"
```

Comment:
I think        ` raise RuntimeError(f"{self!r} is shutdown; no new functions may be executed")`
still fits the line length cutoff here as well?

Reply:
Unfortunately, no.  This one breaks black's draconian line length.